### PR TITLE
185 contact form

### PR DESCRIPTION
**Pull request by mmoravcik**
It will create copy of the email in ContactEmail model.

I moved around some CSS and created common.styl, so please double-check the styling still works across various sections

Also, check and update my wording :smiley_cat:

### CODE REVIEW

**Comment by olasitarska** [comment]
Great job! So helpful. :sparkles: 

I've got one suggestion -- maybe we could make ContactForm a ModelForm and move the email sending logic from the views.py to save() of this ContactForm? I think that would be a lot more neat to keep it there. Let me know what you think :)

**Comment by mmoravcik** [comment]
You probably mean the new `ContactEmail` model I created for saving the sent email.

Initially I wanted just to save the message and sender, but at the end it look like the model is mirroring the `ContactForm`.

I will change it to Modelform

**Comment by mmoravcik** [comment]
@olasitarska have a look now :christmas_tree:

**Comment by mmoravcik** [comment]
@olasitarska have a look now :christmas_tree:

File: core/models.py [hunk 1]
```diff
@@ -174,6 +175,58 @@ def delete(self):
...
+            )
+        except SMTPException:
+            self.sent_successfully = False
+
+        super().save(*args, **kwargs)
+
+    def _get_to_email(self):
+        if self.event and self.event.email:
+            return self.event.email
+        return 'hello@djangogirls.com'
```
Comment (olasitarska) [inline core/models.py]: Should be hello@djangogirls.org

**Comment by olasitarska** [comment]
This looks excellent now! I'll test that locally in an hour :)

**Comment by olasitarska** [comment]
Thanks @mmoravcik, this is awesome! :sparkles:

**Pull request merged**